nx-X11/programs/Xserver/dix/dispatch.c

```diff
@@ -1370,6 +1379,7 @@ ProcQueryFont(register ClientPtr client)
 	    return(BadAlloc);
 	}
 
+	memset(reply, 0, rlength);
```

Comment:
Oh, crap. Thanks. I remember having seen `xcalloc` before now. Yes, it's multiplying its arguments, not initializing with the first parameter. Sorry.